projects/plugins/jetpack/modules/widgets/authors.php

```diff
@@ -34,7 +34,7 @@ public function __construct() {
 			)
 		);
 
-		if ( is_active_widget( false, false, $this->id_base ) || is_active_widget( false, false, 'monster' ) || is_customize_preview() ) {
+		if ( is_customize_preview() ) {
```

Comment:
Untested question: Do we need to add this for the preview if it is in the widget instance? Wouldn't the preview fire off the widget code that enqueues it?

Reply:
Good point. Done in 984e9b0f16144da289a0fb585fe9db1fc7ca561e